### PR TITLE
added London804 profile

### PR DESCRIPTION
**Pull request by London804**

### CODE REVIEW

File: _data/fall_2015/London804.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"emoji": ":wink:",
```
Comment (afeld) [inline _data/fall_2015/London804.json:2]: Ooh, good choice!

Comment (London804) [inline _data/fall_2015/London804.json:2]: Thanks!

File: _data/fall_2015/London804.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+	"introduction": "UX Engineer at Kargo. I love House music."
```
Comment (ericblood1) [inline _data/fall_2015/London804.json:3]: Good intro

Comment (git-ebetts) [inline _data/fall_2015/London804.json:3]: Fantastic!

Comment (joncravotta) [inline _data/fall_2015/London804.json:3]: UX++ !

**Comment by afeld** [comment]
Looks good! :shipit:

**Pull request merged**